bindata/v3.11.0/openshift-controller-manager/cluster-policy-controller-leader-role.yaml

```diff
@@ -0,0 +1,14 @@
+# This role is necessary to create leader lock configmap for upgrades 4.2-> 4.3
+# cluster-policy-controller is split from openshift-controller-manager in 4.3
+# leader lock in openshift-controller-manager NamespaceSecurityAllocationController and in ClusterPolicyController
+# cluster-policy-controller container runs in ns openshift-kube-controller-manager static pod
+# The lock, role, and rolebinding can be removed in 4.4
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: cluster-policy-controller-leader-lock
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["*"]
```

Comment:
enumerate verbs.  We don't use star.

get, list, watch, create, update, delete

Reply:
done